### DISCUSSION TITLE
给作者加油！希望尽快看到 13B 的 plus 版本！

### DISCUSSION THREAD

**Discussion by jamesljl** [ymcui/Chinese-LLaMA-Alpaca#230] [General]
一直在关注作者的中文 Alpaca 模型，并在本地虚拟机部署使用了 13B 的模型，感觉非常棒！
看到 7B Plus 的评测结果后，更期待 13B Plus 的效果了，加油！！！

**Comment by ymcui** [comment]
刚刚发布了Plus-13B版本，欢迎使用。
https://github.com/ymcui/Chinese-LLaMA-Alpaca/releases/tag/v3.1
